vendor/assets/stylesheets/fastruby/styleguide/1-utilities/_variables.scss

```diff
@@ -8,15 +8,21 @@ $white: #f8f8f8;
 $red: #ff0000;
 $yellow: #fdca31;
 $green: #00d242;
+$blue: #1d4ed8;
+$orange: #ff7a00;
 $grey: #323235;
+$soft-grey: #c4c4c4;
 $light-grey: #dadada;
 $lighter-grey: #EAEAEA;
 
 // Measurements
 $image_isotype_height: 70px;
 
 // Typography:
-$font-source-code: "Source Code Pro", monospace;
+$font-source-code: 'Oxygen', sans-serif;
```

Comment:
> @lubc No, the idea behind this PR is to add a css class to the body in the specific project to apply changes only in the STUD project.

@GracielaCesareo  Got it. What about here? Looks like this change will not only affect the SUTD project